google/cloud/bigquery/integration_tests/big_query_read_integration_test.cc

```diff
@@ -34,15 +36,15 @@ using ::testing::Not;
 class BigQueryReadIntegrationTest : public ::testing::Test {
  protected:
   void SetUp() override {
-    connection_options_.enable_tracing("rpc");
+    options_.set<TracingComponentsOption>({"rpc"});
     retry_policy_ =
         absl::make_unique<BigQueryReadLimitedErrorCountRetryPolicy>(1);
     backoff_policy_ = absl::make_unique<ExponentialBackoffPolicy>(
         std::chrono::seconds(1), std::chrono::seconds(1), 2.0);
     idempotency_policy_ = MakeDefaultBigQueryReadConnectionIdempotencyPolicy();
   }
   std::vector<std::string> ClearLogLines() { return log_.ExtractLines(); }
-  BigQueryReadConnectionOptions connection_options_;
+  Options options_;
   std::unique_ptr<BigQueryReadRetryPolicy> retry_policy_;
```

Comment:
My guess is that these `foo_policy_` members are no longer needed since that all now lives in `Options`. Actually, if these policies matter for the tests, they should actually be set in the `options_` data member too.

Fine to handle in a followup PR.